lib/ex_unit/test/ex_unit/assertions_test.exs

```diff
@@ -16,6 +16,10 @@ end
 
 alias ExUnit.AssertionsTest.{BrokenError, Value}
 
+defmodule MinimalMacro do
```

Comment:
Is there a better place to put this?

Reply:
You don't need to define a separate module. If you define it as `defmacro` or `defmacrop` in the current module before you use it you should be fine.